src/pages/EditRequestPage.js

```diff
@@ -90,9 +90,8 @@ const defaultProps = {
     policyTags: {},
 };
 
-function EditRequestPage({betas, report, route, parentReport, policy, session, policyCategories, policyTags}) {
-    const parentReportAction = ReportActionsUtils.getParentReportAction(report);
-    const transaction = TransactionUtils.getLinkedTransaction(parentReportAction);
+function EditRequestPage({betas, report, route, parentReport, policy, session, policyCategories, policyTags, parentReportActions, transaction}) {
+    const parentReportAction = parentReportActions[report.parentReportActionID];
```

Comment:
Maybe we can use lodashGet function here to fetch parentReportAction.

Reply:
Yeah, we could. Ideally, this would be done in the `withOnyx` `selector` for the key `parentReportActions`. When I am able to remove the `withOnyx` nesting, I would like to:

1. Remove this `const parentReportAction `
2. Rename `parentReportActions` prop to `parentReportAction`
3. Update the `transaction` prop to use `parentReportAction`


```
        parentReportAction: {
            key: ({report}) => `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${report ? report.parentReportID : '0'}`,
            selector: (parentReportActions, {report}) => {lodashGet(parentReportActions, [report.parentReportActionID])},
            canEvict: false,
        },
        transaction: {
            key: ({parentReportAction}) => `${ONYXKEYS.COLLECTION.TRANSACTION}${lodashGet(parentReportAction, 'originalMessage.IOUTransactionID', 0)}`,
        },
```

This will be possible once https://github.com/Expensify/react-native-onyx/pull/385 is merged.